source/agora/network/Client.d

```diff
@@ -581,10 +581,10 @@ public class NetworkClient
                     }
                 }
             }
-            if (idx + 1 < this.max_retries) // wait after each failure except last
+            if (idx < this.max_retries) // wait after each failure except last
             {
                 log.dbg("Client.attemptRequest '{}' to all {} connections {}/{} FAILED - wait {} before retry",
```

Comment:
From experience, this log is pretty much useless it does not log which peer we failed to communicate.

Reply:
Will add the address

Reply:
Updated